### PR TITLE
#160451595 Using sendgrid to send email to a user upon successful registration.

### PR DESCRIPTION
**Pull request by Mrshinezee**
#### What does this PR do?
- Configure the project to make use of sendgrid

#### Description of Task to be completed?
- Register on sendgrid and get a sendgrid API key
- With the key, configure your project to send emails.
 
#### How should this be manually tested?
- Clone the repo
- cd into the 'lotus-ah'
- Run 'npm install'
- Use the .env.sample format to create a .env file
- Run 'npm run migrate'
- Run 'npm run start:dev'
     
#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160451595](https://www.pivotaltracker.com/story/show/160451595 "An Email should be sent to a user upon successful registration. User should be able to receive a verification link via email upon successful signup")

### CODE REVIEW

**Review by koiic — APPROVED** [review]
Looking good to me

**Review by Victor-Ugwueze — APPROVED** [review]
Looks good to me.

File: tests/emailControllertest.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import chai from 'chai';
```
Comment (KaiserPhemi) [inline tests/emailControllertest.js]: #### Critical
Rename this file.
All test file should have a `.test` append to the name
i.e. `emailControllertest` should be `emailController.test.js`

File: server/controllers/EmailController.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const key = process.env.SENDGRID_API_KEY;
+const cert = process.env.SECRET;
+const appUrl = process.env.APPURL;
+/**
+ *
+ *
+ * @class UserController
+ */
+class EmailController {
+/**
```
Comment (KaiserPhemi) [inline server/controllers/EmailController.js:15]: #### Nitpick
Reformat the documentation
i.e.
```js
/**
       *
       *
       * @static
       * @param {object} user
       * @return {boolean} result
       * @description Sending email to users that signup
       * @memberof EmailController
       */
```

should be
```js
/**
* @static
* @param {object} user
* @return {boolean} result
* @description Sending email to users that signup
* @memberof EmailController
*/
```

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]

File: server/controllers/EmailController.js (hunk 2 above)
Comment (Mrshinezee) [reply server/controllers/EmailController.js:15]: note

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**